src/internal/router/router.go

```diff
@@ -0,0 +1,25 @@
+package router
```

Comment:
In other cliffhanger projects this package was called routing, you named it router. Any specific reason for that. It's not like you have to rename it, but it might be cool to have some conventions set as well. What do you think?